src/lib/Libutil/pbs_idx.c

```diff
@@ -291,3 +291,24 @@ pbs_idx_free_ctx(void *ctx)
 		ctx = NULL;
 	}
 }
+
+/**
+ * @brief check whether idx is empty and has no key associated with it
+ * 
+ * @param idx[in] - avl index
+ * 
+ * @return int
+ * @retval 1 : idx is empty
```

Comment:
better to keep like `@retval 1 - ...` and `@retval 0 - ...` like other funcs does